drivers/hmis/app/graphql/types/hmis_schema/disability_group.rb

```diff
@@ -0,0 +1,95 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::DisabilityGroup < Types::BaseObject
+    description 'Group of disability records that were collected at the same time'
+
+    field :id, ID, null: false
```

Comment:
which `id` is this?  Won't each disability have different id?

Reply:
It's a concatenated list of all six ids (`1:2:3:4:5:6`). We need an ID for apollo caching, and its helpful for testing too to validate that the correct records are grouped together. I'll add a comment

Reply:
Sounds good.  Thanks!